roles/smallfile/templates/smallfilejob.other_parameters.yaml.j2

```diff
@@ -1,6 +1,14 @@
 
 
 top: {{ smallfile_path }}/smallfile_test_data
+ 
+{% if workload_args.auto_pause is defined %}
+auto-pause: {{workload_args.auto_pause}}
```

Comment:
Missing spaces

Reply:
Oddly there's a mix of spacing that exists in this file, any chance you could standardize the vars in this file to be all spaced correctly?

Reply:
done and pushed